templates/test-build.yml

```diff
@@ -0,0 +1,35 @@
+# <!-- Managed automatically by PreTeXt authoring tools -->
+# (delete the above line to manage this file manually)
+
+name: Test Build
+on:
+    # Runs on pull requests, when a pull request is merged into main,
+    # and when a commit is otherwise pushed to main.
+    push:
```

Comment:
I would rather have this one not automatically run by default either.  I'm worried that a new user will be experimenting, get emails saying an action failed, and get worried.

Reply:
Can we try it like this? If you're right, we can push an update that automatically replaces this with your suggested behavior.

Reply:
And it'll help a lot with support. "Why won't my book build?" With this we just ask for the repo and check out the action and artifact.